packages/block-library/src/gallery/use-image-sizes.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import { get, reduce, map, filter, some } from 'lodash';
```

Comment:
All of these functions except `get` have vanilla JS equivalents.

Reply:
Changed except for one of the `some`s which was over a `{}` instead of `[]` so still lodash for that.